fake-server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const bodyParser = require("body-parser");
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post('/', (req, res) => {

```

Comment:
We should be defining a specific endpoint here, e.g. `/login`

Reply:
yes ok good point